src/TraceEvent/TraceEventNativeMethods.cs

```diff
@@ -464,6 +468,16 @@ internal unsafe struct EVENT_FILTER_EVENT_ID
 
         #endregion
 
+        #region Library functions
+
+        [DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
+        internal static extern IntPtr LoadLibrary([MarshalAs(UnmanagedType.LPWStr)]string lpFileName);
+
+        [DllImport("kernel32.dll", CharSet = CharSet.Unicode)]
```

Comment:
There is no GetProcAddressW. It's ANSI only. So it should be `Charset.ANSI`, `UnmanagedType.LPStr` and you probably want to add `ExactSpelling = true` for good measure (so the CLR doesn't try to find the A/W version)